tools/ci/run_flink_derived_feature_view_example.sh

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2022 The Feathub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+CURRENT_DIR=$(dirname "${BASH_SOURCE-$0}")
+PROJECT_DIR=$(cd "${CURRENT_DIR}/../.."; pwd)
+
+source "${CURRENT_DIR}"/utils.sh
+
+cd "${PROJECT_DIR}/flink-derived-feature-view"
+
+install_deps
+
+chmod 777 data
+docker-compose up -d
+wait_for_port 8081 "Flink Cluster"
+
+python main.py
+docker-compose down
+
+echo "Output: "
```

Comment:
I suppose we will not manually check the output value in the nightly build since it is tedious as the number of tests grows.

It might be simpler to remove this for now. And in the future we can add per-test script to programmatically verify the correctness in the nightly build. Maybe add an issue/TODO?